src/nvim/eval.h

```diff
@@ -113,7 +113,8 @@ enum {
     VV_OPTION_TYPE,
     VV_ERRORS,
     VV_MSGPACK_TYPES,
-    VV_LEN, /* number of v: vars */
+    VV_EVENT,
+    VV_LEN,  // number of v: vars
```

Comment:
You may want to cherry-pick 9a0c6ab50cc37d91574317237035050830f6bf48. I guess your PR will be merged first, it has far less code.